packages/schematics/angular/guard/schema.json

```diff
@@ -15,18 +16,18 @@
     },
     "spec": {
       "type": "boolean",
-      "description": "Specifies if a spec file is generated.",
+      "description": "When true (the default), generates a  \"spec.ts\" test file for the new guard.",
```

Comment:
Nit: would be nice to have the same descriptions for the same flag. 

For instance sometimes, it is `"When true, generates...`` and sometimes it is `When true (the default), generates ..`